homeassistant/components/acmeda/sensor.py

```diff
@@ -1,19 +1,27 @@
 """Support for Acmeda Roller Blind Batteries."""
+from __future__ import annotations
+
 from homeassistant.components.sensor import SensorDeviceClass, SensorEntity
+from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import PERCENTAGE
-from homeassistant.core import callback
+from homeassistant.core import HomeAssistant, callback
 from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
 from .base import AcmedaBase
 from .const import ACMEDA_HUB_UPDATE, DOMAIN
 from .helpers import async_add_acmeda_entities
 
 
-async def async_setup_entry(hass, config_entry, async_add_entities):
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
     """Set up the Acmeda Rollers from a config entry."""
     hub = hass.data[DOMAIN][config_entry.entry_id]
 
-    current = set()
+    current: set[str] = set()
```

Comment:
Same here: I am not 100% sure about the type hint. It seems to be a unique_id, which I assume to be a string.

Reply:
I have traced it to an integer in `aiopulse`: `roller_id, ptr = utils.unpack_int(message, ptr, 6)`
See https://github.com/atmurray/aiopulse/blob/0d6dc539b739c5ea22a8729293526edb3090536c/aiopulse/hub.py#L270-L278